test/read.js

```diff
@@ -52,7 +167,7 @@ describe("Testing jsonapi-client", function() {
       var somePhoto = person.photos[0];
       assert.throws(function() {
         somePhoto.relationships("photographer").add("foobar");
-      }, /Expected Resource, got String/);
+      }, /Expected Resource, got /);
```

Comment:
Try assigning it to a variable first:
```javascript
var a = "foobar";
console.log(typeof a);
```
Before removing the work-in-progress tag I had ~40 commits in this branch poking the tests, I'll try and find the travis build log with the debugging output.